teamcity: failed tests on master: acceptance/TestNodeRestart: result is ambiguous

The following tests appear to have failed:

[#39544](https://teamcity.cockroachdb.com/viewLog.html?buildId=39544):

```
--- FAIL: acceptance/TestNodeRestart (11.946s)
	chaos_test.go:270: pq: result is ambiguous

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/testutils/error.go
+++ b/pkg/testutils/error.go
@@ -57,7 +57,9 @@ func IsPError(pErr *roachpb.Error, re string) bool {
 // some other way. Note that retryable errors may occur event in cases
 // where the SQL execution ran to completion.
 func IsSQLRetryableError(err error) bool {
-	return IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF|commit result is ambiguous)")
+	// Don't forget to update the corresponding test when making adjustments
+	// here.
+	return IsError(err, "(connection reset by peer|connection refused|failed to send RPC|EOF|result is ambiguous)")
 }
 
 // Caller returns filename and line number info for the specified stack

--- a/pkg/testutils/error_test.go
+++ b/pkg/testutils/error_test.go
@@ -1,0 +1,28 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package testutils
+
+import (
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+)
+
+func TestIsSQLRetryableError(t *testing.T) {
+	errAmbiguous := &roachpb.AmbiguousResultError{}
+	if !IsSQLRetryableError(roachpb.NewError(errAmbiguous).GoError()) {
+		t.Fatalf("%s should be a SQLRetryableError", errAmbiguous)
+	}
+}

